Proxy/HTTP/FilterAuthenticatedNonBearerTokens.bambda

```diff
@@ -0,0 +1,56 @@
+/**
+ * Filter when an Authorization header is present, not empty and does not include a traditional bearer token (beginning with "ey")
+ *
+ * @author GangGreenTemperTatum (https://github.com/GangGreenTemperTatum)
+ **/
+
+var configNoFilter = true;
```

Comment:
We suggest that you remove the filter by MIME type logic from this Bambda in order to simplify it. This functionality is also not directly linked to identifying session/authorization tokens.